src/main/kotlin/org/rucca/cheese/model/PostTaskParticipantRequestDTO.kt

```
@@ -0,0 +1,11 @@
+package org.rucca.cheese.model
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.swagger.v3.oas.annotations.media.Schema
+
+/** @param deadline */
+data class PostTaskParticipantRequestDTO(
+    @Schema(example = "null", description = "")
+    @get:JsonProperty("deadline")
+    val deadline: kotlin.Long? = null
+) {}
```

detekt

[warning] The class or object PostTaskParticipantRequestDTO is empty.

(detekt.empty-blocks.EmptyClassBlock)